mtest/libmscore/instrumentchange/add_before-ref.mscx

```diff
@@ -121,12 +121,14 @@
           <gateTime>100</gateTime>
           </Articulation>
         <Channel name="open">
+          <controller ctrl="0" value="0"/>
```

Comment:
Did you change any of the patches in the mixer, by any chance?

Reply:
I don't think so. I've tried to stay away from the mixer so unless I've inadvertently changed something, it shouldn't be something I've done.